release_tester/selenium_ui_test/pages/support_page.py

```diff
@@ -5,6 +5,7 @@
 from selenium_ui_test.pages.navbar import NavigationBarPage
 
 # can't circumvent long lines.. nAttr nLines
+# pylint: disable=C0301 disable=R0902 disable=R0915 disable=R0914
```

Comment:
please use verbatim names instead of C..., as below - so one can better see what actually was disabled.